### PR TITLE
Add router emulation write-up

### PR DESCRIPTION
**Pull request by stulle123**
Hi,

Yet another write-up. Again, I would appreciate your comments.

Thanks,

stulle123

### CODE REVIEW

File: router_emulation/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Router Emulation
+
+As soon as you have the [router's rootfs](../README.md#dumping-nand) you can simply emulate it with Qemu. Steps:
```
Comment (vegantransistor) [inline router_emulation/README.md:3]: "simply" ;-)

File: router_emulation/patched_files/cp__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# decompyle3 version 3.9.1.dev0
+# Python bytecode version base 3.8.0 (3413)
+# Decompiled from: Python 3.8.10 (default, Jun 22 2022, 20:18:18)
+# [GCC 9.4.0]
+# Embedded file name: __init__.py
+# Compiled at: 2022-06-02 13:57:24
+# Size of source mod 2**32: 467 bytes
+__copyright__ = "\nCopyright (c) 2012 CradlePoint, Inc. <www.cradlepoint.com>.\nAll rights reserved.\n\nThis file contains confidential information of CradlePoint, Inc. and your \nuse of this file is subject to the CradlePoint Software License Agreement \ndistributed with this file. Unauthorized reproduction or distribution of \nthis file is subject to civil and criminal penalties. \n"
```
Comment (vegantransistor) [inline router_emulation/patched_files/cp__init__.py:8]: remove header incl. Copyright

File: router_emulation/patched_files/wlan.py
Comment (vegantransistor) [inline router_emulation/patched_files/wlan.py:1]: I would not publish the source code for this file

File: router_emulation/patched_files/link.py
Comment (vegantransistor) [inline router_emulation/patched_files/link.py:1]: I would not publish the source code for this file

**Review by vegantransistor — CHANGES_REQUESTED** [review]

File: router_emulation/patched_files/cp__init__.py (hunk 2 above)
Comment (stulle123) [reply router_emulation/patched_files/cp__init__.py]: Done.

File: router_emulation/patched_files/wlan.py
Comment (stulle123) [reply router_emulation/patched_files/wlan.py:1]: Removed file.

File: router_emulation/patched_files/link.py
Comment (stulle123) [reply router_emulation/patched_files/link.py:1]: Removed file.

**Comment by stulle123** [comment]
@vegantransistor, maybe you can try to reproduce the steps described in `README.md`.

**Pull request merged**